daemon/src/seed.rs

```diff
@@ -88,3 +89,20 @@ impl Default for Seed {
         Self(seed)
     }
 }
+
+impl TryFrom<Mnemonic> for Seed {
+    type Error = anyhow::Error;
+
+    fn try_from(mnemonic: Mnemonic) -> Result<Self> {
+        let entropy = <[u8; 256]>::try_from(mnemonic.to_entropy().as_slice())?;
+        Ok(Self { 0: entropy })
```

Comment:
Lol, does this compile? I had no idea that is valid syntax for referring to fields in tuples!